docs/diagrams/FindGroupSequenceDiagram.puml

```diff
@@ -0,0 +1,79 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":NpcTrackParser" as NpcTrackParser LOGIC_COLOR
```

Comment:
Is the class not called AddressBookParser in our code base? Would it be better to change it to reflect that?